pkg/storage/fs/ocis/node.go

```diff
@@ -342,7 +345,7 @@ func calculateEtag(nodeID string, tmTime time.Time) (string, error) {
 	} else {
 		return "", err
 	}
-	return fmt.Sprintf(`"%x"`, h.Sum(nil)), nil
```

Comment:
hmm? What's the change here? I would rather use the Printf verb. They both yield the same, [see this playground I just put together](https://play.golang.com/p/M-POh8s5rl2)

```golang
package main

import (
	"encoding/hex"
	"fmt"
)

func main() {
	fmt.Printf("raw strings concat:\t%v\nhex verb:\t\t%v",
		`"` + hex.EncodeToString([]byte("1234")) + `"`,
		fmt.Sprintf(`"%x"`, "1234"))
}
```

yields

    raw strings concat:	"31323334"
    hex verb:		    "31323334"
    Program exited.

Reply:
@C0rby said `hex.EncodeToString` is faster

Reply:
probably, but, style? ^^

Reply:
k, for readability I'll go with `fmt.Sprintf(`"%x"`, h.Sum(nil)), nil`